### PR TITLE
New Recipe for Dynamic Component creation

### PR DESCRIPTION
**Pull request by adityanaag3**
### What does this PR do?
Adds a new recipe in the "Composition" menu for Dynamic Component creation

## The PR fulfills these requirements:

[x] Tests for the proposed changes have been added/updated.
[x] Code linting and formatting was performed.

### CODE REVIEW

**Comment by adityanaag3** [comment]
@pozil Created this branch from the `at/codetours` branch instead of `master` by mistake. So pls ignore those files. There is a separate PR for them which Peter is reviewing.

File: src/modules/recipe/compositionDynamic/compositionDynamic.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<template>
+    <ui-card title="CompositionDynamic">
+        <ui-button label="Load Hello Component" onclick={loadComponent}>
+        </ui-button>
+
+        <div class="dynamic-component">
+            <recipe-dynamic-import
```
Comment (pozil) [inline src/modules/recipe/compositionDynamic/compositionDynamic.html:7]: We may want to add a comment about the `recipe-dynamic-import` tag and the `lwc:dynamic` attribute. Users may try to look for a `recipe-dynamic-import` component in our project...

**Review by pozil — APPROVED** [review]
Looking good but you may want to add a comment in the sources.

File: src/modules/recipe/compositionDynamic/compositionDynamic.html (hunk 1 above)
Comment (adityanaag3) [reply src/modules/recipe/compositionDynamic/compositionDynamic.html:11]: Done

**Review by pozil — APPROVED** [review]

**Pull request merged**